e2e/libs/backupstore/minio.py

```diff
@@ -0,0 +1,185 @@
+from backupstore.base import Base
+import os
+import base64
+import json
+import tempfile
+from minio import Minio
+from minio.error import ResponseError
+from urllib.parse import urlparse
+
+class Minio(Base):
+
+    def get_api_client(self, client, core_api, minio_secret_name):
+        secret = core_api.read_namespaced_secret(name=minio_secret_name,
+                                                 namespace='longhorn-system')
+
+        base64_minio_access_key = secret.data['AWS_ACCESS_KEY_ID']
+        base64_minio_secret_key = secret.data['AWS_SECRET_ACCESS_KEY']
+        base64_minio_endpoint_url = secret.data['AWS_ENDPOINTS']
+        base64_minio_cert = secret.data['AWS_CERT']
```

Comment:
I couldn't find where this is used. Consider Removing if not required.

Reply:
Removed. Thank you!